### ISSUE TITLE
Interface setup

### ISSUE THREAD

**Issue by RebeccaGreiner** [amosproj/amos2024ss05-knowledge-graph-extractor#44]
## User story
1. As a developer
2. I need to set up the interface
3. So that we can access the interface through localhost.

## Acceptance criteria
* User interface can be run locally (shown through localhost)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
